subsys/bootloader/cmake/debug_keys.cmake

```diff
@@ -66,12 +74,25 @@ else()
   endif()
 endif()
 
-if ("${CONFIG_SB_PUBLIC_KEY_FILES}" STREQUAL "")
+set(PUBLIC_KEY_FILES "")
+
+# Check if debug public and private keys for key revokation should be generated
+if ("${CONFIG_SB_PUBLIC_KEY_FILES}" STREQUAL "debug")
+  message(WARNING "
+    ----------------------------------------------------------------
+    --- WARNING: Using generated NSIB public keys, it should not ---
+    --- be used for production.                                  ---
```

Comment:
The alignment here is weird. it shoudl be in the middle or wrapped tightly with ----- to match the mcuboot style.

Reply:
I don't see how it is different, and I copied this from mcuboot.